docs/organizations/configuring-default-git-provider-integration-settings.md

```diff
@@ -4,15 +4,15 @@ You can configure the default settings that Codacy uses to integrate with your G
 
 To configure these default settings, go to the Git provider integration page for your organization on the left navigation sidebar.
 
-![Default Git provider integration settings](images/default-git-provider-settings.png)
+![Default Git provider integration settings](images/default-git-provider-settings.png)<!-- TODO CY-6853 -->
```

Comment:
I like your take for updating the screenshots on this page. :+1: Aren't you considering the current version a potential final one?

Reply:
Yes, some of the screenshots are potentially final. I'll flag those as final.